### ISSUE TITLE
Preview panel shows the preview of the first file in a directory for all files

### ISSUE THREAD

**Issue by MatiasConTilde** [gokcehan/lf#1499]
If I want to see the preview of a specific file, the one of the first file in that directory is shown instead

**Comment by MatiasConTilde** [comment]
I downgraded to version 30 and it works fine, if it helps

**Comment by joelim-work** [comment]
I can't reproduce this issue. Can you reproduce this on a default setup with no config file, or at the very least share your configs and more detailed steps to reproduce?

**Comment by CatsDeservePets** [comment]
Closing this issue. 
@MatiasConTilde if this issue still occurs (and can be reproduced), feel free to comment on this issue and I will re-open it again.

**Comment by MatiasConTilde** [comment]
no, I guess in some update in the last two years it fixed itself but I forgot this issue, I'm sorry and thank you!

**Comment by CatsDeservePets** [comment]
It's all good!
